tardis/montecarlo/setup_package.py

```diff
@@ -30,14 +31,16 @@ def get_extensions():
     deps += [os.path.relpath(fname) for fname in glob(
         os.path.join(os.path.dirname(__file__), 'src/randomkit', '*.h'))]
 
-    return [Extension('tardis.montecarlo.montecarlo', sources,
-                      include_dirs=['tardis/montecarlo/src',
-                                    'tardis/montecarlo/src/randomkit',
-                                    np.get_include()],
-                      depends=deps,
-                      extra_compile_args=compile_args,
-                      extra_link_args=link_args,
-                      define_macros=define_macros)]
+    return cythonize(
```

Comment:
@embray we are using package template but have the problem that when `c-files` change that are linked to `pyx-files` `python setup.py develop` does not recompile. @yeganer suggests adding cythonize in a `setup_package.py` to ensure this behaviour. Is this "`astropy`"-conform. Should we include such an example in `package-template`?